istio/README.md

```diff
@@ -0,0 +1,21 @@
+# Istio
+
+## Make a release
+
+To install kURL without Contour Ingress
+
+* Run `make release-kurl-installer`.
+
+To install kURL with Istio along with a test app.
+
+* Run `make release`
```

Comment:
I'd think the README.md would be primarily about installing this kotsapp. So to have the `kots install` and kurl commands here (one for existing cluster, one for embedded clusters). Also with the associated License available for use. 

The `make release` commands seem to be more oriented around contributing to the kotsapp itself.